src/current/cockroachcloud/costs.md

```diff
@@ -189,25 +215,37 @@ Customers will not be charged for data transfer for CockroachDB {{ site.data.pro
 
 For CockroachDB Standard and Advanced, CockroachDB Cloud bills directly to customers based on their usage with a zero-dollar rate during the preview period.
```

Comment:
Standard and Advanced should have separate tabs. I am having a hard time parsing this sentence.